sdk/scheduler/src/main/java/com/mesosphere/sdk/api/PlansResource.java

```diff
@@ -149,6 +151,20 @@ public Response continueCommand(
         return jsonOkResponse(getCommandResult("continue"));
     }
 
+    /**
+     * Interrupts (Pauses) a plan or if specified a phase within a plan.  If a plan/phase is in progress, it
```

Comment:
Opened a ticket since such changes require a different set of people, i.e. reviewers.